src/app/pages/FastBtcPage/components/Deposit/MainScreen.tsx

```diff
@@ -27,7 +27,7 @@ export const MainScreen: React.FC<NetworkAwareComponentProps> = ({
   const { t } = useTranslation();
 
   const { checkMaintenance, States } = useMaintenance();
-  const fastBtcLocked = checkMaintenance(States.FASTBTC);
+  const fastBtcLocked = checkMaintenance(States.FASTBTC_RECEIVE);
```

Comment:
It wasn't introduced in this task but `Receive` button in `Portfolio` is not centered when in maintenance mode. It should be easy to fix.
<img width="1249" alt="image" src="https://user-images.githubusercontent.com/14878713/224716096-b4ceb716-334c-41d9-97a5-cb4a66e76f68.png">